sbt-dumpschema-plugin/src/main/scala/org/virtuslab/akkasaferserializer/model/TypeSymbol.scala

```diff
@@ -0,0 +1,9 @@
+package org.virtuslab.akkasaferserializer.model
+
+sealed trait TypeSymbol
+
+object TypeSymbol {
```

Comment:
Hmmm actually in schema dumps this looks like:

```json
    "typeSymbol": {
      "Class": {

      }
    }
```

I think this can be flattened to just `"typeSymbol": "class"`, right?